### ISSUE TITLE
How can I get test accuracy for each epoch?

### ISSUE THREAD

**Issue by dsy412610** [fchollet/keras#2135]
I can use model.evaluate() to calculate the test accuracy for the last epoch, but how can I get test accuracy for each epoch?

**Comment by dsy412610** [comment]
Can I use test data as validate data?

**Comment by cthacker** [comment]
If you want test accuracy for each epoch, you can set:

> show_accuracy=True

Also to specify the validation data you can do something like this:

> model.fit(X_train, y_train, batch_size=batch_size, nb_epoch=nb_epoch, validation_data=(X_test, y_test), show_accuracy=True)


.

**Comment by dsy412610** [comment]
@cthacker Thank you! I'll try it.
